ci/test_notebooks.sh

```diff
@@ -35,11 +35,11 @@ SKIPNBS=""
 EXITCODE=0
 trap "EXITCODE=1" ERR
 set +e
+# shellcheck disable=SC2044
 for nb in $(find . -name "*.ipynb"); do
```

Comment:
Iterating over loops fed by `find` can be brittle, but this is gnarly to rewrite and seems to work.

Reply:
seesm solid